docs/examples.md

```diff
@@ -1,4 +1,19 @@
 # Examples
 
-Examples using raw Kubernetes YAML, Helm charts, Kustomize and combinations
+### Lifecycle of a Fleet Bundle
+
+To demonstrate the lifecycle of a Fleet bundle, we will use [multi-cluster/helm](https://github.com/rancher/fleet-examples/tree/master/multi-cluster/helm) as a case study.
+
+1. User will create a [GitRepo](./gitrepo-add.md#create-gitrepo-instance) that points to the multi-cluster/helm repository.
+2. The `gitjob-controller` will sync changes from the GitRepo and detect changes from the poll/webhook. With every commit change, the `gitjob-controller` will create a job that clones the git repository, reads content from the repo such as `fleet.yaml` and other manifests, and creates the Fleet [bundle](./cluster-bundles-state.md#bundles).
```

Comment:
> from the poll/webhook

by polling or webhook event.

@jtravee I think this is better, what do you think? And also can we add this link https://fleet.rancher.io/webhook/ to webhook event?

Reply:
Revised per feedback.